docker-compose-non-dev.yml

```diff
@@ -92,10 +92,25 @@ services:
     healthcheck:
       disable: true
 
+  clickhouse:
+    image: clickhouse/clickhouse-server:22.3
+    user: "101:101"
+    env_file: docker/.env-non-dev
+    container_name: clickhouse
```

Comment:
And here please add a prefix `superset`.

Reply:
fixed